docs/platform/howto/manage-project.rst

```diff
@@ -0,0 +1,36 @@
+Create a project
+================
+
+In the `Aiven web console <https://console.aiven.io/>`_, follow these steps to create a new project:
+
+1. Click on **Project** drop down and click on **Create new project**.
+2. Give the new project a name.
+3. When you create a new project, you will need to enable billing for creating new services.
+
+Rename a project
+================
+
+Renaming a project is possible **only** when all the services in the project are powered-off. To rename a project in the `Aiven web console <https://console.aiven.io/>`_:
+
+1. Make sure the respective project is selected from the **Project** drop down.
```

Comment:
Just for clarity and readability, I would reword this to: Select the project from the **Project** drop down.

But up to you!